### PR TITLE
reference new end of sequence component

### PR DESCRIPTION
**Pull request by jwong-d2l**

### CODE REVIEW

File: d2l-sequence-viewer.html [hunk 1]
```diff
@@ -156,9 +150,11 @@
 					></d2l-lesson-header>
 				</span>
 				<span slot="end-of-lesson">
-					<div class="end-of-lesson-holder">
-						<a on-click="showEndOfLesson" class="end-of-lesson">[[localize('endOfSequence')]]</a>
-					</div>
+					<d2l-sequence-end
+						href="[[_sequenceEndHref]]"
+						current-activity="{{href}}"
+						text="[[localize('endOfSequence')]]"
```
Comment (lawong-d2l) [inline d2l-sequence-viewer.html]: Should use the title from the siren object now.

Comment (jwong-d2l) [reply d2l-sequence-viewer.html]: then why do we need this lang term anywhere?

Comment (lawong-d2l) [reply d2l-sequence-viewer.html]: We don't. Not anymore with this new thing. You can delete it here. `endOfSequence` only exists in this repo. Everywhere else I've been changing the word from "Lesson" to "Unit"

Comment (jwong-d2l) [reply d2l-sequence-viewer.html]: maybe we can make a separate pr removing that lang term?

**Pull request merged**